build/optaplanner-build-parent/pom.xml

```
@@ -68,6 +68,7 @@
 
     <schema.filename.benchmark>benchmark-with-import.xsd</schema.filename.benchmark>
     <schema.filename.solver>solver.xsd</schema.filename.solver>
+    <sonar.coverage.exclusions>**/swingui/**/*</sonar.coverage.exclusions>
```

Just for a clarity that we exclude Java files in that path (both should work):
```suggestion
    <sonar.coverage.exclusions>**/swingui/**/*.java</sonar.coverage.exclusions>
```